lib/Github/Api/Issue/Comments.php

```diff
@@ -41,18 +41,24 @@ public function configure($bodyType = null)
      *
      * @link https://developer.github.com/v3/issues/comments/#list-comments-on-an-issue
      *
-     * @param string $username
-     * @param string $repository
-     * @param int    $issue
-     * @param int    $page
+     * @param string    $username
+     * @param string    $repository
+     * @param int       $issue
+     * @param int|array $page
      *
      * @return array
      */
     public function all($username, $repository, $issue, $page = 1)
     {
-        return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/issues/'.rawurlencode($issue).'/comments', [
-            'page' => $page,
-        ]);
+        if (is_array($page)) {
+            $parameters = $page;
+        } else {
+            $parameters = [
```

Comment:
Can you add a deprecation warning here, indicating that it's deprecated to pass a different type other than array to the `$page` parameter and that an array will be required in the future?

`@trigger_error('message', E_USER_DEPRECATED);`

Reply:
Thank you for your reply.
I added a @trigger_error and a comment with reference to the following code.
https://github.com/KnpLabs/php-github-api/blob/6e843e62160bb586ece922026226ea6a00fc1f74/lib/Github/Api/Issue/Assignees.php#L61